### ISSUE TITLE
astra_data_source_available_regions lists only classic DB regions

### ISSUE THREAD

**Issue by harishdesetti1206** [datastax/terraform-provider-astra#24]
Hi there,

Please provide the following details with your issue report.

### Terraform Version
Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.

### Affected Resource(s)
Please list the resources as a list, for example:
- astra_database
- astra_keyspace

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files
```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### Debug Output
Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

While creating a DB in Datastax Astra(Azure) using Terraform  an error is dsiplayed. Error: cloud provider and region combination not available:

Note: I'm using the Astra from Microsoft azure subscription as SaaS.
I created from the DataStax Astra DB for Apache Cassandra - Astra Service Plan - Monthly (The Logo in Orange colour)

### Expected Behavior
What should have happened?
An DB instance should be created in the Astra portal.
### Actual Behavior
What actually happened?
An error is returned for terraform apply command saying "cloud provider and region combination not available:"
### Steps to Reproduce

Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

Console Output:
Error: cloud provider and region combination not available: AZURE/westus2

### References
Related issue:
- https://github.com/datastax/terraform-provider-astra/issues/19

Terraform version: 0.14.10
Astra provider version: 0.0.4-pre

**Comment by phact** [comment]
Should be fixed in v1.0.4+

**Comment by harishdesetti1206** [comment]
Great @phact the issue is fixed.. i used the version 1.0.6 and it worked like charm .. Thanks .!
